Plugins/ServerPlugin.cpp

```diff
@@ -1,8 +1,9 @@
+#ifndef _WIN32_WINNT
+#define _WIN32_WINNT 0x0600  // at least windows vista required for grpc

```

Comment:
As we discussed, please don't let this get out of hand. Move it somewhere common if it becomes an issue.

Reply:
I won't yet, but I don't think there will ever be a need to. The server can really only have an implementation defined in one location because C++ does not have partial classes. Although it's possible the logic could be separated out into additional server plugin sources if it gets too big, none of those will actually need to include that server proto header. So this is likely the only file that will ever be including GRPC and need that defined.